docs/ref/models/expressions.txt

```diff
@@ -371,7 +372,8 @@ SQL that is generated. Here's a brief example::
                 expression,
                 distinct='DISTINCT ' if distinct else '',
                 output_field=IntegerField(),
-                **extra)
+                **extra,
```

Comment:
I put together a small example to try and test this.. and it did work.
What leads you to believe the trailing comma is a syntax error? The **extra?

```python
class BaseDummy(object):
    def __init__(self, posarg, **args):
        for k,v in args.items():
            print("K:{} Value:{}".format(k, v))
            
class Count(BaseDummy):

    def __init__(self, expression, distinct=False, **extra):
        super(Count, self).__init__(
            expression,
            distinct='DISTINCT ' if distinct else '',
            output_field="",
            **extra,
        )

if __name__ == "__main__":
    c = Count(r'.*', distinct=True, a=1, b=2, c=3)



```